### ISSUE TITLE
Support for Logstash 5

### ISSUE THREAD

**Issue by nadeemmar** [yokawasa/logstash-output-azure_loganalytics#1]
Hello,

I tried installing this plugin but am getting the below error which indicates the plugin isn't supported on this version of logstash. Are there plans to upgrade it? 

> Validating logstash-output-azure_loganalytics
> Installing logstash-output-azure_loganalytics
> Plugin version conflict, aborting
> **ERROR: Installation Aborted, message: Bundler could not find compatible versions for gem logstash-core:**
>   In snapshot (Gemfile.lock):
>     logstash-core (= 5.2.2)
> 
>   In Gemfile:
>     logstash-core-plugin-api (>= 0) java depends on
>       logstash-core (= 5.2.2) java
> 
>     logstash-output-azure_loganalytics (>= 0) java depends on
>       logstash-core (< 3.0.0, >= 2.0.0) java
> 
>     logstash-core (>= 0) java
> 
> Running `bundle update` will rebuild your snapshot from scratch, using only
> the gems in your Gemfile, which may resolve the conflict.

**Comment by yokawasa** [comment]
My apologies for responding to you this late. I'm looking into this.

**Comment by koksan83** [comment]
Hi,

Experiencing similar issue:
```
Validating logstash-output-azure_loganalytics
Installing logstash-output-azure_loganalytics
Plugin version conflict, aborting
ERROR: Installation Aborted, message: Bundler could not find compatible versions for gem "logstash-core":
  In snapshot (Gemfile.lock):
    logstash-core (= 5.5.1)

  In Gemfile:
    logstash-core-plugin-api (>= 0) java depends on
      logstash-core (= 5.5.1) java

    logstash-output-azure_loganalytics (>= 0) java depends on
      logstash-core (< 3.0.0, >= 2.0.0) java

    logstash-core (>= 0) java

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

**Comment by JoaquimFreitas** [comment]
Hi,
Experiencing the same issue:

```
Validating logstash-output-azure_loganalytics
Installing logstash-output-azure_loganalytics
Plugin version conflict, aborting
ERROR: Installation Aborted, message: Bundler could not find compatible versions for gem "logstash-core":
  In snapshot (Gemfile.lock):
    logstash-core (= 5.6.2)

  In Gemfile:
    logstash-core-plugin-api (>= 0) java depends on
      logstash-core (= 5.6.2) java

    logstash-output-azure_loganalytics (>= 0) java depends on
      logstash-core (< 3.0.0, >= 2.0.0) java

    logstash-core (>= 0) java

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```
All goes on the version of the `logstash-core` dependency declared in the gem file.

**Comment by Publish-Interactive** [comment]
Is this still broken in version 5? I don't see any new commits...

**Comment by KiZach** [comment]
There is a forked version here https://github.com/arthurtoper/logstash-output-azure_loganalytics that works with both 5x and 6x

**Comment by yokawasa** [comment]
I merged #5, now it supports logstash5.X and 6.X. 
Please install `logstash-output-azure_loganalytics-0.2.1`
see also [this](https://github.com/yokawasa/logstash-output-azure_loganalytics/releases/tag/v0.2.1)
Thanks
